src/SDPoptimize.jl

```diff
@@ -117,30 +117,9 @@ function generate_grid(model::SPModel, param::SDPparameters)
 end
 
 """
-Value iteration algorithm to compute optimal value functions in
-the Decision Hazard (DH) as well as the Hazard Decision (HD) case
-
-Parameters:
-- model (SPmodel)
-    the DPSPmodel of our problem
-
-- param (SDPparameters)
-    the parameters for the SDP algorithm
-
-- display (Bool)
-    the output display or verbosity parameter
-
-
-Returns :
-- value_functions (Array)
-    the vector representing the value functions as functions of the state
-    of the system at each time step
-
+Try to construct a StochDynProgModel from an SPModel
 """
-function sdp_optimize(model::SPModel,
-                  param::SDPparameters,
-                  display=true::Bool)
-
+function build_sdpmodel_from_spmodel(model::SPModel)
```

Comment:
I had the same idea on my branch sdp_getters2 and used it in get_control function.
The problem with this architecture is that we have to do the conversion every time we call a function that needs a StochDynProgmodel (for example when we want to call forward_simulation after sdp_optimize). Couldn't we store the StochDynProgmodel in the general SPmodel after the first conversion?